Dockerfile

```diff
@@ -4,15 +4,15 @@ RUN apk add --no-cache git musl-dev \
     && go get -v -u github.com/googlecloudplatform/gcsfuse
 
 # build exhibitor
-FROM maven:3.6-alpine as builder
+FROM maven:3.8-eclipse-temurin-11-alpine as builder
 
 WORKDIR /exhibitor
 COPY . .
 
 RUN mvn -B -DskipTests=true install \
     && mvn -B -nsu -f exhibitor-standalone/src/main/resources/buildscripts/standalone/maven/pom.xml package
 
-FROM openjdk:8-jdk-alpine
+FROM eclipse-temurin:11-jdk-alpine
```

Comment:
Why use `eclipse-temurin` and not `openjdk`?

Reply:
I guess you mean Docker repo for [OpenJDK](https://hub.docker.com/_/openjdk)? The repo uses same OpenJDK distribution as the one I use, see [here](https://github.com/docker-library/openjdk/blob/198895da839835656064d35ed7b480bfa4587f91/11/jdk/bullseye/Dockerfile#L52). I needed an `alpine` flavor, and the official `adoptium` (`eclipse-temurin`) distribution provides one.

For more information about the difference between the OpenJDK distributions, there is a excellent **stackoverflow** [thread](https://stackoverflow.com/questions/52431764/difference-between-openjdk-and-adoptium-adoptopenjdk) explaining that...